storage/url/url.go

```diff
@@ -291,7 +312,7 @@ func (u *URL) MarshalJSON() ([]byte, error) {
 
 // HasGlob reports whether if a string contains any wildcard chars.
 func (u *URL) HasGlob() bool {
```

Comment:
```suggestion
func (u *URL) IsWildcard() bool {
```
At this stage, I think we can rename this function since it also checks raw mode. Not sure about naming :D

Reply:
I am also not sure about the name, so if we can agree on a name, then I will change it 😄

Reply:
My choice is `IsWildcard()`

Reply:
`HasGlob()` is changed to `IsWildcard()`